test/e2e/runner/evidence.go

```diff
@@ -0,0 +1,320 @@
+package main
```

Comment:
I'm not reviewing this file line-by-line as I assume it comes as-is from some cherry-pick.
Let me know if that's not the case, and I'll do through it more thoroughly.